src/js/components/Drop/Drop.js

```diff
@@ -16,7 +16,19 @@ class Drop extends Component {
 
   originalFocusedElement = document.activeElement;
 
-  dropContainer = getNewContainer();
+  constructor(props) {
+    super(props);
+
+    this.dropContainer = getNewContainer();
+
+    // prevents mouse events from propagating upward to elements behind the drop.
+    // this prevents a nested drop from closing its parent when the user clicks
+    // inside it, since a 'nested' drop is technically an unrelated sibling in the
+    // DOM due to React portals.
+    this.dropContainer.addEventListener('mousedown', ev => {
```

Comment:
I think we are supposed to add event listeners in `componentDidMount()` and not in the constructor?

Reply:
Can do. Didn't research a best practice.

I should also probably unregister it on unmount, too 😅